test/cmocka/src/debugability/macros.c

```diff
@@ -48,19 +48,20 @@ static void test_debugability_macros_declare_log_entry(void **state)
 		LOG_LEVEL_CRITICAL,
 		"Message",
 		TRACE_CLASS_DMA,
+		1,
 		1
 	));
 	const char *should_be_eq =
 		"__attribute__((section(\".static_log.\""
 		" \"LOG_LEVEL_CRITICAL\"))) "
-		"static const struct { uint32_t level; uint32_t component_id; "
-		"uint32_t params_num; uint32_t line_idx; uint32_t file_name_len; "
-		"const char file_name[sizeof(\"src/debugability/macros.c\")]; "
-		"uint32_t text_len; const char text[sizeof(\"Message\")]; } "
-		"log_entry = { 1(6 << 24)152sizeof(\"src/debugability/macros.c\")"
-		"\"src/debugability/macros.c\"sizeof(\"Message\")\"Message\" }";
+		"static const struct { uint32_t level; uint32_t component_class; "
+		"uint32_t has_ids; uint32_t params_num; uint32_t line_idx; "
+		"uint32_t file_name_len; uint32_t text_len; const char "
+		"file_name[sizeof(\"src/debugability/macros.c\")]; "
+		"const char text[sizeof(\"Message\")]; } "
+		"log_entry = { 1(6 << 24)1153sizeof(\"src/debugability/macros.c\")"
+		"sizeof(\"Message\")\"src/debugability/macros.c\"\"Message\" }";
```

Comment:
I were to change this myself, but forgot.
As we already use
```
"aa;bbb;"
"ccc;dddd;"
```
and expect it to become `"aa;bbb;ccc;dddd;`,
we can go 1 step further and make it
```
"aa;"
"bbb;"
"ccc;"
"dddd;"
```
Something like
```
"static const struct {"
" uint32_t level;"
" uint32_t component_class;"
...
```

This can be even more beneficial in `log_entry = {` body
Consider the following
```
"log_entry = { "
/*level       */"1"
/*component_id*/(6<<24)
...
/*file_name   */\"src/debugability/macros.c\"
/*text        */"\"Message\"
```
This would make debugging in the future considerably easier.

Reply:
adjusted